src/data/attribute.rs

```diff
@@ -129,7 +130,42 @@ pub fn store_metadata(metastruct: &Struct) {
     let metadata = process_metadata(metastruct, String::new());
     for (key, value) in metadata {
         let attr = format!("{KUADRANT_NAMESPACE}\\.{key}");
+        // stored into host_property: wasm.kuadrant.{key}
+        // example: wasm.kuadrant.identity.anonymous is how it's stored!
+        // but users would write the predicate: !auth.identity.anonymous
+        // two problems:
+        // - 1/ auth.identity doesn't resolve
+        // - 2/ the value is the string "true"
+
+        // struct User {
+        //   foo: bool,
+        //   bar: float, // 1 != 1.0
+        //   name: String,
+        // }
+        //
+        // Admin can store this:
+        // authorino: export auth.identity.user.foo = expression("auth.user != null") // {"foo": true, "bar": 123, "name": "dd"}
+        // Or that:
+        // authorino: export auth.identity.foo = expression("auth.user.long.ass.path.to.some.member.within.foo")
+        // authorino: export auth.identity.user.bar = 123
+        // authorino: export auth.identity.user.name = "dd"
+
+        // predicate: !auth.identity.user.foo && auth.identity.user.bar != 443.0
+        //      => properties [["auth", "identity", "user", "foo"], ["destination", "port"]]
```

Comment:
where are those `destination` and `port` coming from?

Reply:
... and json numbers are floats: `1` in JSON is `1_f64` using rust notation. It is not an integer.